### ISSUE TITLE
Toggleable 3D for when armor is worn

### ISSUE THREAD

**Issue by TheNamesJames** [tr7zw/3d-Skin-Layers#147]
Love the mod, but I've had a recurring issue with this mod and other armor mods. No conflicts or anything, but the 3D parts often enough clip through armor from mods (especially helmets). I am aware that there is a config option that can shrink 3D voxel sizes, but there are helmets that are clipped through at levels that make this mod redundant. Would it be possible to have a config option to disable the 3D effect when a piece of armor is equipped?

**Comment by tr7zw** [comment]
I assume this is about modded helmets that for some reason are smaller than the vanilla ones?

**Comment by meeprdib** [comment]
Yeah, two mods I've noticed issues with are Iron's Spells and Spellbooks and Alex's Caves, but I'm sure there's a lot more out there.

**Comment by TheNamesJames** [comment]
> I assume this is about modded helmets that for some reason are smaller than the vanilla ones?

Yeah that's it. I was having an issue with Deeper & Darker and a few other mods I believe but I just lived with it. Recently I started modifying the modpack I was working on a year ago, updating things, and I found out that Deeper & Darker fixed its helmet model. I also noticed that you added a compatibility mode to the config which is awesome :)
